templates/etc/haproxy/haproxy-default.cfg.j2

```diff
@@ -45,3 +45,6 @@ defaults
 {% if haproxy_default_unique_id_format is defined and haproxy_default_unique_id_format | length %}
     unique-id-format {{ haproxy_default_unique_id_format }}
 {% endif %}
+{% if haproxy_default_load_server_state_from_file is defined and haproxy_default_load_server_state_from_file|length %}
+    load-server-state-from-file {{ haproxy_default_load_server_state_from_file | default('global') }}
```

Comment:
Not sure you a need to set `default()` as the statement only apply if `haproxy_default_load_server_state_from_file` is defined.

Reply:
I agree. Will remove it.